addOns/oast/src/main/java/org/zaproxy/addon/oast/ExtensionOast.java

```diff
@@ -275,6 +279,33 @@ public OastPayload registerAlertAndGetOastPayload(Alert alert) throws Exception
         return null;
     }
 
+    public String getBoastPayload(boolean isNew) {
+        String payload = "";
+        try {
+            payload = isNew ? boastService.getNewPayload() : boastService.getLastestPayload();
+        } catch (IOException e) {
+            LOGGER.error("Could not generate new payload.", e);
+        }
+        return payload;
+    }
+
+    public String getInteractshPayload(boolean isNew) {
+        String payload = "";
+        try {
+            payload =
+                    isNew
+                            ? interactshService.getNewPayload()
+                            : interactshService.getLatestPayload();
+        } catch (URIException | InteractshException e) {
+            LOGGER.error("Error during interactsh register: {}", e.getMessage(), e);
+        }
+        return payload;
+    }
+
+    public String getCallbackPayload(boolean isNew) {
+        return isNew ? callbackService.getNewPayload() : callbackService.getLatestPayload();
+    }
```

Comment:
This should not be here, the services should be accessed directly, also, the common behaviour should be provided through `OastService`.

Reply:
okay. Should I add it in the respective service class as a public function ?

Reply:
Is this fine now ? Please check

Reply:
The duplicated code in the service implementations should be moved to `OastService`.

Reply:
![Screenshot from 2023-02-14 18-47-39](https://user-images.githubusercontent.com/87857234/218750332-461df069-9c3f-4b95-8516-05fd1d060c4c.png)


> The help should mention the context menu. Some tests should be added.

okay I am working on all the issues. Thankyou.